### ISSUE TITLE
Make an icon

### ISSUE THREAD

**Issue by Adubbz** [Adubbz/Gag-Order#1]
As per title.

**Comment by Valeri0p** [comment]
Idea for the art maker: since the name is "gag order", and it blocks the browser from working, something simple like a :globe_with_meridians: with an unzipped :zipper_mouth_face: should do the trick[... ](http://www.freestencilgallery.com/wp-content/uploads/2017/08/Emoji-Zipper-Mouth-Stencil-thumb.jpg).
Thanks again for your program :D

**Comment by gingerbeardman** [comment]
I can do this for you if you let me know the specs

**Comment by Adubbz** [comment]
Thank you! I would be interested to see a polished version of Valeri0p's suggestion. This one has been floating around but I'd like something more refined https://cdn.discordapp.com/attachments/453536450739437578/453612986528169985/a66xrtfz.png

**Comment by Valeri0p** [comment]
> Thank you! I would be interested to see a polished version of Valeri0p's suggestion. This one has been floating around but I'd like something more refined https://cdn.discordapp.com/attachments/453536450739437578/453612986528169985/a66xrtfz.png

Thanks for your appreciation ☺ BTW, isn't that icon the one used in @tumGER 's pack?

**Comment by Adubbz** [comment]
Yeah I believe he compiled Gag Order with his own custom icon for his pack

**Comment by gingerbeardman** [comment]
128 x 128 pixels?
On black, solid colour, pattern image, transparent?

**Comment by Adubbz** [comment]
256x256, not too fussed about what the background is (whatever you think looks nice), however it's probably best to avoid transparency

**Comment by gingerbeardman** [comment]
Where is this being displayed?

**Comment by Adubbz** [comment]
On the homebrew menu, so both on a white background (the cards) and dark blue

**Comment by gingerbeardman** [comment]
OK I'm on it.

**Comment by AnnsAnns** [comment]
> Thanks for your appreciation ☺ BTW, isn't that icon the one used in @tumGER 's pack?

Hi, I kinda forgot to actually post that image to the issue, yeah I created that image for my SDFiles project since I didn't want to include GagOrder in such a raw form ^^'

It was a pretty dirty gimp edit and never aimed to be the official one
